replay-protected-sighash.md

```diff
@@ -36,7 +36,7 @@ The proposed digest algorithm computes the double SHA256 of the serialization of
 1. nVersion of the transaction (4-byte little endian)
 2. hashPrevouts (32-byte hash)
 3. hashSequence (32-byte hash)
-4. outpoint (32-byte hash + 4-byte little endian)
```

Comment:
The intention here was to move it to a more generic chapter (h3) as the concept of 'outpoint' is reused in several other places in the following chapters.  It is not immediately clear when reading about hashPrevouts what an outpoint is. This clarifies.